### ISSUE TITLE
[UI Framework] Material Design

### ISSUE THREAD

**Issue by RXWE** [swsnu/swpp2019-team6#25]
https://material-ui.com/

**Comment by RXWE** [comment]
Material Design을 중점적으로 사용하여 UI 구성을 진행하도록 하겠습니다.

**Comment by RXWE** [comment]
## Installation

1. Core

```
// with npm
$ npm install @material-ui/core

// with yarn
$ yarn add @material-ui/core
```

2. SVG Icons
```
// with npm
$ npm install @material-ui/icons

// with yarn
$ yarn add @material-ui/icons
```

3. Roboto Font

`<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap" />`

OR

```
$ npm install typeface-roboto --save
```

WITH

`import 'typeface-roboto';`

**Comment by deploy-soon** [comment]
3. `Roboto Font` 는 프로젝트 전반적으로 적용되도록 해야하는건가요?

**Comment by RXWE** [comment]
>     3. `Roboto Font` 는 프로젝트 전반적으로 적용되도록 해야하는건가요?

저도 정확히는 파악하지 못했는데, TypoGraphy 컴포넌트에 글꼴이 적용되게 하려면 사용되는 곳에 저 링크를 넣어줘야 한다고 이해하긴 했어요. 좀 더 자세히 알게되면 또 적겠습니다

**Comment by deploy-soon** [comment]
네 `index.js` 혹은 `index.css`에 넣어두고 프론트 처음 접근할 때 글꼴 다운 받도록 하면 될 것 같아요
